Code/GraphMol/FMCS/MaximumCommonSubgraph.cpp

```diff
@@ -656,8 +657,8 @@ std::pair<std::string, RWMol*> MaximumCommonSubgraph::generateResultSMARTSAndQue
   // Generate result's SMARTS
 
   // create molecule from MCS for MolToSmarts()
-  RWMol *mol = new RWMol();
-  const RingInfo *ri = QueryMolecule->getRingInfo();
+  RWMol* mol = new RWMol();
```

Comment:
I have to say, I'm not a fan of this formatting.
```
RWMol* m, mol;
```

Doesn't do what you might expect.  The *,& qualifier is a qualifier for the variable not the type unless you use a typedef.

Reply:
I just double checked, and clang-format formats these lines differently:

    RWMol* mol = new RWMol();

and:

    RWMol* m, mol;

Reply:
Odd, I'm only getting that style when I use --style=Mozilla

clang-format foo.cpp --style=Mozilla
```
void
f()
{
  RWMol* m = new RWMol(m);
  RWMol *m, mol;
}
```

clang-format foo.cpp --style=file (.clang-format from rdkit)
```
void f() {
  RWMol *m = new RWMol(m);
  RWMol *m, mol;
}
```

Reply:
something is probably messed up with my vs-code config. I will take a look.
But, again, this will hopefully be dealt with when I re-run clang-tidy after this PR is merged